src/components/edit-member.js

```diff
@@ -0,0 +1,117 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+
+class EditMember extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
```

Comment:
One of the notes in the assignment is to avoid using state in React components. How would you approach the edit screen in such a way that all of the state is managed by Redux?

Reply:
@adamstep, should I make changes in the code or just explain it to you here?

Reply:
If you have time to make the changes, that would be better. If not, an explanation is fine.

Reply:
I am explaining it here then, will also try to make changes in code as well if will get some time today. 

To use redux to manage form's data, the first thing we need is currently editing member's data in redux's state. We will change the `memberIdToEdit` to `memberToEdit` in redux state and will dispatch a action like `START_EDIT_MEMBER` to set `memberToEdit`. 

In the edit screen, all the data will come from this `memberToEdit` in redux state and all the changes in form inputs will dispatch actions to change the data in redux state (I mean, will make new state).

Doing so, this single form will add a lot of actions in the reducer and it will be better to use multiple reducers.

We can also use something like **redux-form**, haven't used it but read about it sometime ago.